plugin/storage/grpc/shared/grpc_client_test.go

```diff
@@ -200,9 +201,11 @@ func TestGRPCClientGetTrace_NoTrace(t *testing.T) {
 }
 
 func TestGRPCClientGetTrace_StreamErrorTraceNotFound(t *testing.T) {
+	s, _ := status.FromError(spanstore.ErrTraceNotFound)
+
 	withGRPCClient(func(r *grpcClientTest) {
 		traceClient := new(grpcMocks.SpanReaderPlugin_GetTraceClient)
-		traceClient.On("Recv").Return(nil, spanstore.ErrTraceNotFound)
+		traceClient.On("Recv").Return(nil, s.Err())
```

Comment:
I think we should write a test that goes through client and server, not just mocking the client.

Reply:
Agree.  I can't say I have time right now to knock it out.  Perhaps if i find some time in the next week or two I'll dig into it.